### ISSUE TITLE
[selectmenu] Should the text inside `<option>`s be selectable?

### ISSUE THREAD

**Issue by mfreed7** [openui/open-ui#687]
It feels like option text (within the popped-up listbox) should be user-selectable, so they can copy it to the clipboard. But perhaps there's an a11y or other reason not to allow that. I just wanted to ask the question explicitly.

**Comment by scottaohara** [comment]
hmm. good question.  not something presently possible with select element options, and not the most straight forward to do with other interactives, like buttons (i can mouse down outside of a button and then select the text by mouse moving/keyboard moving the text caret that way.  but directly mouse down / dragging on a button other interactive controls that listen for a mouse up/down click event would have conflicts?  maybe a larger issue than a11y

**Comment by mfreed7** [comment]
Thanks - very good points. And thanks for pointing out that you can't currently do this on `<select>`, or at least in any way that I know of. I hadn't noticed that.

Perhaps the best would be a **non-**`!important` UA rule that says `option {user-select:none}`? Then if there's actually a use case, developers could override that, but in most cases, text isn't selectable?

**Comment by scottaohara** [comment]
i think that makes sense.  the returned value for the selectmenu in the triggering element _should_ behave similarly to button, in that someone could click/drag or keyboard shift key press and left or right arrow key to include the visible text of that trigger.  that'd be one way someone could copy the currently chosen option at least.

i dunno.  it's a feature that i could see people wanting when they want it.  but otherwise, not really caring too much about since it's not a common thing that can be done (easily)?

**Comment by brechtDR** [comment]
Couldn't this actually be beneficial for A11Y in some manner? Being able to copy your options for when you're having a messaging chat with a person and you want to ask which option to take. If it's not harmful in any way for A11Y, I think it could be a nice-to-have for some people.
I'm thinking about this case from a personal perspective, helping a less tech-savy elderly person out with a form submission via chat.

**Comment by josepharhar** [comment]
I don't have a strong opinion, but I think that we should leave it selectable because other clickable/selectable elements like `<button>` are user-select:auto. However, I can't seem to actually select the text in the `<button>`s with a mouse.
It also looks like the selectmenu implementation in chromium behaves the same way: There is no user-select:none rule, but you can't select the text with a mouse.

**Comment by josepharhar** [comment]
I'm not sure if this needs any text in the explainer, but it will at least be in the user agent stylesheet and I'm adding an implementation with WPTs here: https://chromium-review.googlesource.com/c/chromium/src/+/4370763
